tools/pyang/pyang_plugins/openapi.py

```diff
@@ -64,6 +67,11 @@
     "415": {"description": "Unsupported Media Type"},          
 }
 verb_responses = {}
+verb_responses["rpc"] = {
```

Comment:
RPC is not a verb, it is an operation defined through YANG's RFC statement, the only supported operation for RPC as per RESTCONF specification is POST. But the set of HTTP status codes for RPC POST operation is different from the other POST operations (on the data nodes such as container, list, etc). 

verb_responses dictionary contains a set of status codes for each of the RESTCONF verbs since RPC is a deviation as explained and it supports just a POST and we have added a dedicated entry for it.